test/integration/query-interface.test.js

```diff
@@ -421,6 +421,21 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
         expect(table).to.not.have.property('fruitId');
       });
     });
+
+    it('shows a reasonable error message when column is missing', function() {
+      var self = this;
+      var Users = self.sequelize.define('_Users', {
+        username: DataTypes.STRING
+      }, { freezeTableName: true });
+
+      return Users.sync({ force: true }).then(function() {
```

Comment:
Can be simplified a lot with just ` promise.to.be.rejectedWith(Error)` assertion api